### ISSUE TITLE
Bug: application shell layout flex bug

### ISSUE THREAD

**Issue by DamianLion** [SAP/fundamental#1199]
**Describe the bug**

> Is this issue related to a specific component?
application shell layout 

> What version of the Fiori Fundamentals are you using?
1.4.1

> What front-end framework are you implementing in, e.g., Angular, React, etc.?

native - https://codepen.io/DeltaM/pen/YBKZJN
vue - https://codepen.io/DeltaM/pen/YBKZJN

**To Reproduce**

https://codepen.io/DeltaM/pen/YBKZJN

**Expected behavior**

correct flexbox layouting

**Screenshots**

![grafik](https://user-images.githubusercontent.com/1820250/51460686-75ad5000-1d5c-11e9-850e-4aaabab877a7.png)

**Desktop (please complete the following information):**
 - OS: MacOS
 - Browser Chrome, Firefox

**Additional context**

A fix for out solution was to set the flex to `1` instead of `1 auto`

```
.fd-app__main {
  -webkit-box-flex: 1;
  -ms-flex: 1;
  flex: 1;
}
```

**Comment by xak** [comment]
This is worth investigating but the ideal implementation is **not** to put components directly into the `app__main` container but to use [a `page` container](https://sap.github.io/fundamental/layouts/page-layout.html).

Then within the `page` you can use `sections which will provide proper margins and padding. See the markup of the [sidebar demo page](https://sap.github.io/fundamental/demo-pages/sidebar-app-demo.html) to see the ideal implementation.

**Comment by DamianLion** [comment]
@xak thanks for your reply but also the `sidebar demo page` you sent me has the same error of responsiveness

![grafik](https://user-images.githubusercontent.com/1820250/52208529-441aa580-2881-11e9-9449-594deb04df51.png)

**Comment by pavan-sokke-nagaraj** [comment]
@saad-mo In which release will this bug be fixed? Currently `fiori-fundamentals` is in `1.4.5`  and the bug still there.
Thanks

**Comment by saad-mo** [comment]
@pavan-sokke-nagaraj This ticket is already part of our current development milestone. Unfortunately, we do not have a release date at this point, but it would not be released until a couple more weeks as we are still in the early stages of the milestone.

**Comment by ChristianKienle** [comment]
@saad-mo I think this is fixed in 1.4.9. I can no longer reproduce it and have closed the related fundamental vue issue.
